### ISSUE TITLE
Eggs Setting `stable` to false, where it's set to true in config

### ISSUE THREAD

**Issue by ebebbington** [nestdotland/eggs#103]
## 🐛 Bug Report

My config file:

```json
{
  "releaseType": "minor",
  "ignore": [
    ".git"
  ],
  "name": "data_format_converter",
  "description": "Data Format Converter for Deno: XML, JSON, CSV",
  "version": "1.2.0",
  "stable": true,
  "homepage": "https://github.com/ebebbington/data-format-converter",
  "files": [
    "./mod.ts",
    "./dfc-logo.png",
    "./LICENSE",
    "./src/*.ts",
    "./README.md"
  ],
  "entry": "./mod.ts",
  "checkAll": false,
  "unlisted": false
}
```

Result from eggs:
```
Edwards-MacBook-Pro:data-format-converter edwardbebbington$ eggs publish
- [INFO] Source files are formatted.
- [INFO] Tests passed successfully.
- [INFO] No errors detected when installing the module.
[INFO] The resulting module is: 
        - name: "data_format_converter",
        - version: "1.3.0",
        - description: "Data Format Converter for Deno: XML, JSON, CSV",
        - repository: "https://github.com/ebebbington/data-format-converter",
        - unlisted: false,
        - stable: false,
```

Notice `stable` is false

## To Reproduce

Unsure how someone else would reproduce this :S But all the info is above or below

## Expected behavior

The `stable` prop. should be `true`

## Actual Behavior

The stable prop is being set to false

## Environment

- Eggs version: 0.3.1
- Deno version: 1.4.6
- Operating system and version: MacOS High Sierra 10.13.6

**Comment by ebebbington** [comment]
Think it'd be good to add extra tests around this too If it's possible

**Comment by SteelAlloy** [comment]
The `stable` field is not one of the allowed fields, I'm sorry I didn't see it.
This was removed in `v0.3.0` https://github.com/nestdotland/eggs/releases/tag/0.3.0
You have to use `unstable` instead, which is `false` by default.
But that doesn't explain why stable is `false` here.

> 
> Think it'd be good to add extra tests around this too If it's possible

Yes, this is true. We are actively working on version 1.0 and the CLI will take a completely different form so I'm focusing on testing this new version.

**Comment by ebebbington** [comment]
I only mentioned the extra tests as i 'thought' it was a bug

I'm happy to close this issue unless you wanted to keep it open

**Comment by SteelAlloy** [comment]
`stable` should be true anyway by default, you can keep it open 😄

**Comment by ebebbington** [comment]
👍

**Comment by SteelAlloy** [comment]
fixed in #105
